doc/developer_guide/testing.rst

```diff
@@ -44,6 +44,7 @@ To debug specific tests (with flag [testHere]) from the Visual Studio IDE, you c
     }
 
 Select the testRunner.exe in the startup item dropdown menu and you can start debugging tests with the specified flag.
+You can provide multiple arguments like `"[testHere][testHere2]"` to run all test that have both flags, or with comma separation `"[testHere],[testHere2]"` to run all tests with the first and/or the second flag.
```

Comment:
Consider listing all valid flags here. Or to describe how all flags can be listed.